lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -618,6 +618,17 @@ ifeq "$(USESWIFTDRIVER)" "1"
 	endif
 endif
 
+#----------------------------------------------------------------------
+# Set the clang that Swift uses for linking
+#----------------------------------------------------------------------
+ifeq "$(SWIFT_DRIVER_CLANG_EXEC)" ""
+	export SWIFT_DRIVER_CLANG_EXEC = $(CC)
```

Comment:
What is the semantics of `export` here?

Reply:
It's supposed to add it as an environment variable to child-processes. I was hoping it would get picked up by the `$(SWIFTC)` invocations below so that I wouldn't need to set it on the environment on all of them and things would just work, but it did not.